### PR TITLE
[FLINK-25619] Init flink-table-store repository

### PR DESCRIPTION
**Pull request by JingsongLi**
Create:
- README.md
- NOTICE LICENSE CODE_OF_CONDUCT
- .gitignore
- maven tools
- releasing tools
- github build workflow
- pom.xml

Refer to https://github.com/apache/flink-ml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+## <a name="build"></a>Building the Project
+
+Run the `mvn clean package` command.
+
+Then you will find a JAR file that contains your application, plus any libraries that you may have added as dependencies to the application: `target/<artifact-id>-<version>.jar`.
+
+## <a name="contributing"></a>Contributing
+
+You can learn more about how to contribute in the [Apache Flink website](https://flink.apache.org/contributing/how-to-contribute.html). For code contributions, please read carefully the [Contributing Code](https://flink.apache.org/contributing/contribute-code.html) section for an overview of ongoing community work.
```
Comment (LadyForest) [inline README.md:15]: Nit: **on** the website is more accurate.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,697 @@
...
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
```
Comment (LadyForest) [inline pom.xml:25]: It seems like Flink uses Tab as indentation for the pom file (but code indentation is four spaces). Should we align?

**Review by LadyForest — COMMENTED** [review]
Hi, @JingsongLi thanks for your contributions. 

LGTM, in general, and I just left some comments towards wording and indentation.

File: pom.xml (hunk 2 above)
Comment (JingsongLi) [reply pom.xml:25]: Flink old style all with tab, later only changed the code to 4 spaces, did not change the pom, I think we can directly use the space.

Comment (JingsongLi) [reply pom.xml]: I will use 4 spaces

**Comment by LadyForest** [comment]
LGTM :)

**Pull request merged**